### ISSUE TITLE
Create CKEditor plugin for Materialize Icon set

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1323]
Fork this -- http://ckeditor.com/addon/fontawesome
plugin doesn't work outright but their demo site is awesome. Might be a good thing to reverse engineer into a MaterializeCSS plugin for it's icon set though there's so many icons it'll be a pain to map them all. Still, this plugin has a search function against the icons displayed w/ real time updating, it's pretty slick even if it's not working in our setup.

**Comment by btopro** [comment]
plugin doesn't work outright but their demo site is awesome. Might be a good thing to reverse engineer into a MaterializeCSS plugin for it's icon set though there's so many icons it'll be a pain to map them all. Still, this plugin has a search function against the icons displayed w/ real time updating, it's pretty slick even if it's not working in our setup.

**Comment by btopro** [comment]
not needed if we have hax / #1855
